src/main/java/com/exasol/containers/LiteralExasolDockerImageReference.java

```diff
@@ -62,4 +62,16 @@ public boolean hasDockerImageRevision() {
     public String toString() {
         return this.reference;
     }
+
+    @Override
+    public String getSuffix() {
+        // TODO Auto-generated method stub
+        return null;
+    }
+
+    @Override
+    public boolean hasSuffix() {
+        // TODO Auto-generated method stub
+        return false;
```

Comment:
Can we remove TODO from these two methods? Or if there is anything still needs to be done, we can also create a ticket.